miso-web/src/main/webapp/pages/bulkEditDilutions.jsp

```diff
@@ -53,6 +53,8 @@
     <p>To fill all empty cells in a column with the value of your selected cell(s), <b>double-click</b> the square in the bottom right of your selected cell(s).
       <br/>To fill a variable number of columns with the value of your selected cell,  <b>click</b> the square in the bottom right of your 
       filled-in selected cell and <b>drag</b> up or down. All selected columns will be filled in.
+      <br/>To fill down a column with values following an incremental (+1) pattern, either select two adjacent cells in a column and then drag 
```

Comment:
@djcooke made a good point with the wording - could you please move the "either" a few words over, to "and then either drag down,..."? /neckbeard